packages/main/src/components/SplitterLayout/SplitterLayout.stories.mdx

```diff
@@ -0,0 +1,157 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Button } from '@ui5/webcomponents-react/dist/Button';
+import { DocsHeader } from '@docs/DocsHeader';
+import { SplitterLayout } from '@ui5/webcomponents-react/dist/SplitterLayout';
+import { SplitterElement } from '@ui5/webcomponents-react/dist/SplitterElement';
+import { FlexBox } from '@ui5/webcomponents-react/dist/FlexBox';
+import { SegmentedButton } from '@ui5/webcomponents-react/dist/SegmentedButton';
+import { SegmentedButtonItem } from '@ui5/webcomponents-react/dist/SegmentedButtonItem';
+import { Label } from '@ui5/webcomponents-react/dist/Label';
+import { useState, useEffect } from 'react';
+
+<Meta
+  title="Layouts & Floorplans / SplitterLayout"
+  component={SplitterLayout}
+  subcomponents={{ SplitterElement }}
+  argTypes={{
+    children: { control: { disable: true } }
+  }}
+  args={{ style: { width: '100%', height: '800px' } }}
+/>
+
+<DocsHeader />
+
+<br />
+
+## Example
+
+<Canvas>
+  <Story name="Default">
+    {(args) => {
+      return (
+        <SplitterLayout {...args}>
+          <SplitterElement>
+            <Button style={{ width: '100%' }}>Content 1</Button>
+          </SplitterElement>
+          <SplitterElement minSize={200}>
+            <Button style={{ width: '100%' }}>Content 2 (minSize: 200)</Button>
+          </SplitterElement>
+          <SplitterElement>
+            <Button style={{ width: '100%' }}>Content 3</Button>
+          </SplitterElement>
+          <SplitterElement resizable={false}>
+            <Button style={{ width: '100%' }}>Content 4 (not resizable)</Button>
+          </SplitterElement>
+          <SplitterElement>
+            <Button style={{ width: '100%' }}>Content 5</Button>
+          </SplitterElement>
+          <SplitterElement>
+            <Button style={{ width: '100%' }}>Content 6 (size: 200px)</Button>
+          </SplitterElement>
+        </SplitterLayout>
```

Comment:
Although this works as intended I would try to keep it a bit simpler. I tried to move around elements and directly hit the min-size or the non-resizable element which feels buggy for the end user.
Maybe only add one element that has a min Size and a max size, the other elements can resize freely?
And add a second story for non-resizable elements?

Reply:
Done.